src/frontend/src/handler/describe.rs

```diff
@@ -139,8 +140,16 @@ pub fn handle_describe(context: OptimizerContext, table_name: ObjectName) -> Res
         Some(rows.len() as i32),
         rows.into(),
         vec![
-            PgFieldDescriptor::new("Name".to_owned(), TypeOid::Varchar),
-            PgFieldDescriptor::new("Type".to_owned(), TypeOid::Varchar),
+            PgFieldDescriptor::new(
+                "Name".to_owned(),
+                DataType::VARCHAR.to_oid(),
+                DataType::VARCHAR.type_len(),
+            ),
+            PgFieldDescriptor::new(
+                "Type".to_owned(),
+                DataType::VARCHAR.to_oid(),
```

Comment:
> Why we need this?

Like this case, we need the oid and type len of VARCHAR, but we don't need to store a type. @BowenXiao1999

Reply:
I think we can implement to_oid on DataType? 🤔 

Use this to reduce code: https://docs.rs/enum_dispatch/latest/enum_dispatch/